src/website/app/demos/routes.js

```diff
@@ -17,58 +17,86 @@
 /* @flow */
 export default {
   button: {
+    description:
+      'A Button should be used whenever you need to trigger an action in your app. The color of the button should be chosen according to the intent of the action.',
     slug: 'button',
     title: 'Button'
   },
   card: {
+    description:
+      'Cards are used to group different kinds of elements with actions attached. Use a Card to contain content that might not fit neatly into a table row or grid cell.',
     slug: 'card',
     title: 'Card'
   },
   'card-block': {
+    description:
+      'CardBlock is used to help lay out content that’s not a title or an image in the body of the Card.',
     slug: 'card-block',
     title: 'CardBlock'
   },
   'card-image': {
+    description:
+      'Use a CardImage to reinforce the intent of the Card. Images shouldn’t be used alone in a Card, but should be paired with a call to action and/or a CardTitle.',
     slug: 'card-image',
     title: 'CardImage'
   },
   'card-title': {
+    description:
+      'Use a CardTitle when you need a consistently styled heading for your Card. Use a subtitle to provide supporting information for the data displayed in the Card.',
     slug: 'card-title',
     title: 'CardTitle'
   },
   dropdown: {
+    description:
+      'Use Dropdown for extra functionality, not for primary actions, since the options are hidden from the user until interaction.',
     slug: 'dropdown',
     title: 'Dropdown'
   },
   icon: {
+    description:
+      'Icons can symbolize actions and objects in your interace. Use icons in combination with labels to help users more quickly process your UI.',
     slug: 'icon',
     title: 'Icon'
   },
   link: {
+    description:
+      'Links should be used when the browser location will change, and should be clear about where the user will navigate.',
     slug: 'link',
     title: 'Link'
   },
   menu: {
+    description:
+      'A Menu should be used whenever you need to display a concise list of options to your user.',
     slug: 'menu',
     title: 'Menu'
   },
   'menu-divider': {
+    description:
+      'MenuDividers are used to visually separate MenuGroups or individual MenuItems to establish hierarchy in your Menu.',
     slug: 'menu-divider',
     title: 'MenuDivider'
   },
   'menu-group': {
+    description:
+      'MenuGroups are used to group conceptually related elements, and to hint at other available, related options.',
     slug: 'menu-group',
     title: 'MenuGroup'
   },
   'menu-item': {
+    description:
+      'A MenuItem represents an option in a Menu. They can be used to trigger actions or navigate to a new location.',
     slug: 'menu-item',
     title: 'MenuItem'
   },
   popover: {
+    description:
+      'Use Popovers to contain content providing clarification for other interface elements. Popovers can be used to implement other custom behaviors or widgets.',
```

Comment:
Popovers display supporting content when your user interacts with it's associated trigger.

Reply:
I changed a couple of these from the source material (usually the usage section) to be more "action-oriented" to be more engaging to potential users. So instead of just saying what the component does, you phrase it more like "use component x like so".

Maybe something like "Use Popovers to display supporting content..."?

Reply:
I was thinking about someone reading this in their search results and defining the component felt more helpful in that context than something action oriented. The search result blurbs are brief too, so leaving out the words, "use" and "to" seemed attractive.